ed/idlpatches/payment-request.idl.patch

```diff
@@ -0,0 +1,53 @@
+From d00f84ad858cc457d0e894218931ef6b28cc6d4d Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Mon, 17 May 2021 13:47:50 +0200
+Subject: [PATCH] Restore PaymentAddress and AddressErrors interface
+
+Still used in Contact API:
+https://github.com/WICG/contact-api/pull/39
+
```

Comment:
wouldn't it make more sense to instead patch the Contact API and Basic Card?

My particular worry is to figure out what will trigger the removal/update of that patch.

Reply:
Possibly, but whereever I put it, updates to the specs are unlikely going to trigger a merge conflict for the patch, so I suspect that we will have to track the issues referenced in the patch no matter what to determine whether the patch may be dropped.

Reply:
hmm... surely, when  https://github.com/WICG/contact-api/pull/39  gets merged, this will create a conflict if we have an equivalent here?

Reply:
after discussion with @tidoust, and given the unclear future of Basic Card, reverting to the latest consistent state sounds like the better approach

Reply:
Discussed live. Generic idea is to stick to "past" and not implement updates in advance. We'll leave this as is for now.